build/commands/lib/config.js

```diff
@@ -879,6 +879,15 @@ Config.prototype.update = function (options) {
   if (options.target) {
     this.buildTarget = options.target
   }
+
+  if (this.use_goma && this.gomaServerHost) {
+    if (!this.extraNinjaOpts.find(val => typeof val === 'string' && val.startsWith('-j'))) {
+      this.extraNinjaOpts.push('-j', this.gomaJValue)
+      console.log('using goma with j value of ' + this.gomaJValue + ' at ' + this.gomaServerHost)
+    } else {
+      console.log('using goma with manual j value at ' + this.gomaServerHost)
```

Comment:
Is this about passing a custom j value? We can probably think of an env var to override our formula.

Reply:
yes, this for passing custom `-j` value, because currently it's impossible to do this - hardcoded `gomaJValue` is prioritised over `--ninja=j:xxx`.

Reply:
I think this change would be easier to understand if `gomaJValue` -> `defaultGomaJValue`

Reply:
> `gomaJValue` -> `defaultGomaJValue`

agree. renamed.